test/utils.py

```diff
@@ -615,3 +617,86 @@ def wait_pid_disappear(file, target_pid):
                 found = False
                 break
     assert not found
+
+
+def copy_data(dst, file_paths):
+    for path in file_paths:
+        shutil.copy(path, dst)
+
+
+def get_version(tt_cmd, tmpdir):
```

Comment:
1. Why do you need `tmpdir`? `tt_cmd` is an absolute path, and `version` request does not depend on where it will be running.
2. It seems that we have two functions - one for Tarantool `get_tarantool_version` and this new one. And it would be wonderful to have the same response format, at least. What are the benefits of returning `boolean` status and `patch` number?

Reply:
Agree. `get_version()` has been removed, all other `tt` helpers moved to the `utils.py` with applied `get_tarantool_version()`.